src/brackets.js

```diff
@@ -91,6 +91,7 @@ define(function (require, exports, module) {
     require("editor/CSSInlineEditor");
     require("project/WorkingSetSort");
     require("search/QuickOpen");
+	require("search/QuickOpenHelper");
```

Comment:
Nitpick: Align indent with others (I think this line has tabs instead of spaces)